### PR TITLE
Add guide on forwarding certificates on Kubernetes NGINX Ingress

### PR DESCRIPTION
**Pull request by rctay**

### CODE REVIEW

**Comment by rctay** [comment]
Hi @anthonyvetter, I've shifted this from the blog section to the guide as discussed. Also put in the copywriter's edits.

**Comment by anthonyvetter** [comment]
Looks like you have some spell check failing. FYI, `Cloudflare` and probably `PEM` are okay to add to `custom_dict.txt`. `lient` looks to be a typo maybe from `[C]lient` on line 21. I would just change that unless there is a good reason to have the work formatted like that. `ssl_client_escaped_cert` is an interesting one, and could be interpreted as a bug on our side. This corresponds to the instance of the word on line 83 "[^ssl_client_escaped_cer]" where you are creating footnotes. I have been trying to fix this with regex by ignoring single words within square brackets, but I haven't found the right incantation yet. I will continue to work on this. Alternatively, if you wanted to move away from the footnotes model, you could replace this and the CloudFlare footnote with 
```This corresponds to the NGINX variable [`$ssl_client_escaped_cert`](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#var_ssl_client_escaped_cert).``` 
This passes spelling tests. Your call.

**Comment by rctay** [comment]
Thanks for taking a look, @anthonyvetter.

I've put in the fixes, curiously spellcheck is OK with another footnote ID `nginx_ssl_cert_docs` instead of `ssl_client_escaped_cert ` so I went with that.

I want to point out for the `[C]lient` one that came from the copywriter, I think the square bracket is used to indicate the case differs from the quoted source, ala <https://writingcommons.org/article/inserting-or-altering-words-in-a-direct-quotation/>. So to fix the spellcheck, I wrapped it in a spellcheck ignore. A little unsightly but it's one spot so I hope that's ok.

File: content/guides/kubernetes/nginx-ingress-certificate-forwarding.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Forwarding Client Certificates with NGINX Ingress"
+description: >
+  A look at annotations to configure Kubernetes NGINX Ingress for forwarding client certificates
+date: 2021-05-17
+tags:
+  - Kubernetes
+team:
+  - Ray Chuan Tay
+---
```
Comment (anthonyvetter) [inline content/guides/kubernetes/nginx-ingress-certificate-forwarding.md:10]:
```suggestion
topic:
  - Kubernetes
  - Messaging and Integration
---
```

**Comment by anthonyvetter** [comment]
I didn't actually know you could wrap in a spellcheck ignore :) And that is a good reason for the formatting. Okay the only other things I see:
* Needs a Topic in the front matter so the blog actually shown up in the correct lists. I added a suggestion to the file. 
* I added a card/issue to track your content to our [content project board](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7#card-65668361). I linked this issue to your PR.
Otherwise this LGTM. Add a topic and, if you are okay with the Netlify staging preview, I will get it merged ASAP.

**Pull request merged**